modules/dnn/test/test_model.cpp

```diff
@@ -273,7 +273,7 @@ TEST_P(Test_Model, Keypoints_pose)
 #endif
 
     Mat inp = imread(_tf("pose.png"));
-    std::string weights = _tf("onnx/models/lightweight_pose_estimation.onnx", false);
+    std::string weights = _tf("lightweight_pose_estimation_201912.onnx", false);
```

Comment:
Keep `onnx/models`.

Model is not updated yet through `download_models.py` - this not fully automated process.
ETA 10-20min - need to scan SHA of several Gb (on all build machines).